### PR TITLE
banco azteca icon

### PR DESCRIPTION
**Pull request by alfaro28**
## Why are you adding this icons?
I'm adding/updating this icon(s) because Banco Azteca icon is missing and we want to use it for our payment app

#### Help us identify yourself
- [ ] I'm working/collaborating with the brand directly and they have provided the icons.
- [x] I'm associated with the brand and I've read all the brand icon’s guidelines.
- [ ] I'm an individual and I've read all the brand icon’s guidelines.

Link to the brand guidelines: <Link>

## Checklist to add new icons
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I have attached a screenshot comparison with the example icon provided in guidelines
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

## Attach a screenshot of the icon along side the example Visa icon

Captura desde 2024-04-29 19-01-42

#### If the icons are intended for use by Shopify, please provide the following info:
Who are you working with at Shopify? (avoid adding personal details, provide github handle(preferred) or first name and last name)

What's the expected date of this change to deploy on Shopify?

### CODE REVIEW

**Review by Lovedanihonjin — CHANGES_REQUESTED** [review]
There is black background on your SVG file. Please fix and outline your stroke. Thank you!

**Comment by alfaro28** [comment]
Hi @Lovedanihonjin could you please review again?

File: db/payment_icons.yml [hunk 1]
```diff
@@ -2210,3 +2210,7 @@
   name: qliro
   label: Qliro
   group: other
+-
+  name: banco_azteca
```
Comment (Lovedanihonjin) [inline db/payment_icons.yml:2214]: We changed our guidelines. Please remove _ from the name.

**Review by Lovedanihonjin — CHANGES_REQUESTED** [review]
SVG image is missing border, and border radius is incorrect. Please refer to out appearance guidelines: https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#appearance

Screenshot 2024-05-09 at 9 18 10 AM
Screenshot 2024-05-09 at 9 18 50 AM

**Comment by alfaro28** [comment]
Hi @Lovedanihonjin could you please review again?

**Comment by alfaro28** [comment]
Hi @Lovedanihonjin , I would like to know an estimation of when is this PR going to be merged?

**Comment by alfaro28** [comment]
Hi @Lovedanihonjin is there anything we still need to do to have this PR merged?

**Comment by alfaro28** [comment]
Hi @Lovedanihonjin I would just like some visibility on why this PR is not being merged despite being approve, and some visibility on an approximate date of release

**Comment by dannye0231** [comment]
Hi @alfaro28 , are you able to check the conflicts for this PR?  I am unable to merge it due to an error that needs to be resolved via the command line

**Comment by alfaro28** [comment]
Hi @dannye0231 , I fixed the conflicts

**Comment by alfaro28** [comment]
Hi @dannye0231 , do you have an estimated date for the next release?

**Comment by dannye0231** [comment]
Hi @alfaro28 , we are planning to do another run this Wednesday.  You should see it merged into Prod by the end of this week.

**Pull request merged**